python/cudf/cudf/tests/test_datetime.py

```diff
@@ -1278,12 +1278,8 @@ def test_datetime_reductions(data, op, dtype):
 @pytest.mark.parametrize("dtype", DATETIME_TYPES)
 def test_datetime_infer_format(data, dtype):
```

Comment:
This test verifies if our string-to-time conversions work properly until nano-second time resolution. Can you add another set of tests that drop the `timezone="UTC"` parameter?

Reply:
Sure, I'll add back that testing in this test